components/3js/animatedMesh.tsx

```diff
@@ -122,6 +122,19 @@ function AnimatedMesh({
   return <mesh ref={meshRef} geometry={geometry} material={material} />;
 }
 
+function EnvironmentLighting() {
```

Comment:
There's no diff between this `<EnvironmentLighting />` and the one in animatedAsterisk. `import` and `export` it from the other.

Reply:
Colors differ.